aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -261,6 +267,71 @@ public static CreateDbInstanceRequest createDbInstanceRequest(
                 .build();
     }
 
+    static Instant stringToInstant(String restoreTimeString) {
+        if (com.amazonaws.util.StringUtils.hasValue(restoreTimeString)) {
+            try {
+                return LocalDateTime.parse(
+                                restoreTimeString,
+                                DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
+                        )
+                        .atOffset(ZoneOffset.ofHours(0))
+                        .toInstant();
+            } catch (Exception e) {
```

Comment:
Please catch a format exception: `Exception` selector is way too non-discriminative.